src/ops/ops.go

```diff
@@ -195,6 +196,27 @@ func (o *ops) Reboot() error {
 	return nil
 }
 
+func (o *ops) SetBootOrder(device string) error {
+	_, err := o.ExecPrivilegeCommand(o.logWriter, "test", "-d", "/sys/firmware/efi")
+	if err != nil {
+		o.log.Info("efi not supported")
+		return nil
+	}
+
+	o.log.Info("Setting efibootmgr to boot from disk")
+	_, err = o.ExecPrivilegeCommand(o.logWriter, "mount", device+"2", "/mnt")
+	if err != nil {
+		o.log.Errorf("Failed to mount device %s, err: %s", device+"2", err)
+		return err
+	}
```

Comment:
Wouldn't it make the coreos-installer fail since it checks the device for [busy partitions](https://github.com/coreos/coreos-installer/blob/7b689344a1f9f9692345a41d9dc81f810251c9f6/src/install.rs#L123-L142)?

Reply:
Doesn't the installer complete before this happens?

Reply:
@eranco74  @tsorya do you know?